src/main/java/com/okta/tools/aws/settings/AwsINIConfiguration.java

```diff
@@ -0,0 +1,875 @@
+/*
```

Comment:
🤔 before calling me on this. I admit that this is a copy/paste from Apache Commons Configuration2. I tried to extend their class and it wasn't open to extension in this manner. I will look into contributing an extension point to their project so that we can later replace this with something much cleaner.

Reply:
It's literally this code:
https://github.com/apache/commons-configuration/blob/trunk/src/main/java/org/apache/commons/configuration2/INIConfiguration.java

With some changes I'll highlight in the review.